OneDrive/Área de Trabalho/OficinaMecanica/src/main/java/com/mycompany/telas/TelaUsuario.java

```diff
@@ -4,17 +4,134 @@
  */
 package com.mycompany.telas;
 
+import java.sql.*;
+import com.mycompany.dal.ModuloConexao;
+import javax.swing.JOptionPane;
+
 /**
  *
  * @author lucas
  */
 public class TelaUsuario extends javax.swing.JInternalFrame {
 
+    Connection conexao = null;
+    PreparedStatement pst = null;
+    ResultSet rs = null;
+
     /**
      * Creates new form TelaUsuario
      */
     public TelaUsuario() {
         initComponents();
+        conexao = ModuloConexao.conector();
+    }
+
+    private void consultar() {
+        String sql = "select * from tbusuarios where iduser=?";
+        try {
+            pst = conexao.prepareStatement(sql);
+            pst.setString(1, txtUsuId.getText());
+            rs = pst.executeQuery();
+            if (rs.next()) {
+                txtUsuNome.setText(rs.getString(2));
+                txtUsuLogin.setText(rs.getString(4));
+                txtUsuFone.setText(rs.getString(3));
+                txtUsuSenha.setText(rs.getString(5));
+                cboUsuPerfil.setSelectedItem(rs.getString(6));
+            } else {
+                JOptionPane.showMessageDialog(null, "Usuário inexistente");
+                txtUsuNome.setText(null);
+                txtUsuLogin.setText(null);
+                txtUsuFone.setText(null);
+                txtUsuSenha.setText(null);
+            }
+        } catch (Exception e) {
```

Comment:
```suggestion
        } catch (Exception exception) {
```
evitar abreviações para não causa ambiguidade em leitura